src/main/java/com/alkemy/ong/service/CommentService.java

```diff
@@ -28,4 +32,40 @@ public CommentDto save(CommentDto dto) {
          public List<CommentBodyDto> getAll() {
             return commentMapper.commentEntityListToCommentBodyDtoList(commentRepository.findAllByOrder());
         }
+
+    @Transactional
+    public ResponseEntity<?> delete(Authentication aut, String id) {
+        if ( verifyId(aut, id)) {
+            CommentEntity entity = commentRepository.getById(id);
+            entity.setSoftDelete(true);
+            commentRepository.save(entity);
+            return new ResponseEntity<>(HttpStatus.ACCEPTED);
+        } else {
+            throw new ResponseStatusException(HttpStatus.NOT_FOUND,
+                    "There is no News with the entered Id");
```

Comment:
Shouldn't it say there is no comment?
also I wonder if we implement ParamNotFoundException